README.md

```diff
@@ -1,8 +1,77 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
 
+# Xinyu Wang . Jarvis Consulting
+Driven and analytical professional with a bachelor's level education in electrical engineering from Carleton University.
```

Comment:
please use first person narrative for the description. 

please see this example
https://github.com/jarviscanada/jarvis_data_eng_mathew

Reply:
this section could be longer and more descriptive. talk to Amullya if you need English help